Computationally intensive statistics in Rust (#1025)
online-ml/river: CONTRIBUTING.md

$ conda activate river
```

> You need a `Rust` compiler you can install it by following this [link](https://www.rust-lang.org/fr/tools/install)

Reviewer: ```suggestion
You need a Rust compiler which you can install [here](https://www.rust-lang.org/fr/tools/install)
```
